test/e2e/e2e.go

```diff
@@ -1488,6 +1488,71 @@ var _ = ginkgo.Describe("e2e ingress gateway traffic validation", func() {
 	})
 })
 
+// This test validates OVS exports NetFlow data from br-int to an external collector
+var _ = ginkgo.Describe("e2e br-int NetFlow export validation", func() {
+	const (
+		ovnNs                     string = "ovn-kubernetes"
+		netFlowCollectorContainer string = "netflow-collector"
+		ciNetworkName             string = "kind"
+	)
+
+	ginkgo.AfterEach(func() {
+		// tear down the NetFlow container
+		if cid, _ := runCommand("docker", "ps", "-qaf", fmt.Sprintf("name=%s", netFlowCollectorContainer)); cid != "" {
+			if _, err := runCommand("docker", "rm", "-f", netFlowCollectorContainer); err != nil {
+				framework.Logf("failed to delete the netFlow collector test container %s %v", netFlowCollectorContainer, err)
+			}
+		}
+	})
+
+	ginkgo.It("Should validate NetFlow data of br-int is sent to an external gateway", func() {
+		var (
+			ciNetworkFlag = "{{ .NetworkSettings.Networks.kind.IPAddress }}"
+		)
+
+		// start the NetFlow collector container that will receive data
+		_, err := runCommand("docker", "run", "-itd", "--privileged", "--network", ciNetworkName, "--name", netFlowCollectorContainer, "cloudflare/goflow", "-kafka=false")
+		if err != nil {
+			framework.Failf("failed to start NetFlow collector test container %s: %v", netFlowCollectorContainer, err)
+		}
+		// retrieve the container ip of the NetFlow collector container
+		netFlowCollectorIp, err := runCommand("docker", "inspect", "-f", ciNetworkFlag, netFlowCollectorContainer)
+		if err != nil {
+			framework.Failf("failed to start NetFlow collector test container: %v", err)
+		}
+		// trim newline from the inspect output
+		netFlowCollectorIp = strings.TrimSuffix(netFlowCollectorIp, "\n")
+		if ip := net.ParseIP(netFlowCollectorIp); ip == nil {
+			framework.Failf("Unable to retrieve a valid address from container %s with inspect output of %s", netFlowCollectorContainer, netFlowCollectorIp)
+		}
+
+		framework.Logf("Setting OVN_NETFLOW_TARGETS environment variable value to NetFlow collector IP %s", netFlowCollectorIp)
+		framework.RunKubectlOrDie(ovnNs, "set", "env", "daemonset/ovnkube-node", "-c", "ovnkube-node", "OVN_NETFLOW_TARGETS="+netFlowCollectorIp)
```

Comment:
this is cool didn't know you could do this :+1:

Reply:
yep, I randomly discovered last week, I used to just resort to kubectl edit ds, add envvar.